leftwm-core/src/handlers/command_handler.rs

```diff
@@ -369,9 +373,9 @@ fn swap_tags(state: &mut State) -> Option<bool> {
         std::mem::swap(&mut state.workspaces.get_mut(hist_a)?.tag, &mut temp);
         // Update dock tags and layouts.
         state.update_static();
-        state
-            .layout_manager
-            .update_layouts(&mut state.workspaces, state.tags.all_mut());
```

Comment:
What is the point of the `update_layouts` method? Is it just so the `tag.layout` and `workspace.layout` are kept the same? I guess my refactoring makes this obsolete with the new layout manager.